core/src/main/scala/scala/pickling/internal/DefaultPicklerRegistry.scala

```diff
@@ -10,14 +10,16 @@ import scala.pickling.spi.{PicklerRegistry, RuntimePicklerGenerator}
 
 /** Default pickle registry just uses TrieMaps and delgates behavior to a runtime pickler generator. */
 final class DefaultPicklerRegistry(generator: RuntimePicklerGenerator) extends PicklerRegistry with RuntimePicklerRegistryHelper {
-  type PicklerGenerator = FastTypeTag[_] => Pickler[_]
-  type UnpicklerGenerator = FastTypeTag[_] => Unpickler[_]
+
+  type PicklerGen = FastTypeTag[_] => Pickler[_]
```

Comment:
I see here, I was inconsistent in my existential usage (reality, I think I was just moving code around and never cleaned it up :) ).